node/_tools/test.ts

```diff
@@ -57,7 +57,7 @@ for await (const path of testPaths) {
         "--unstable",
         "--no-check",
         "--v8-flags=" + v8Flags.join(),
-        requireTs,
+        "--compat",
```

Comment:
Required to avoid errors with forked processes which start complaining because of the `.ts` extension - would need the require helper file to be `.mjs` for it to work... or `--compat` (which opted for here).

Reply:
I prefer `.mjs` here. It's more important to check that non-compat deno can consume `std/node` correctly in my view